frontend/public/components/storage-class-form.tsx

```diff
@@ -1000,6 +1020,22 @@ export class StorageClassForm_ extends React.Component<
             </span>
           </div>
 
+          {expansionFlag && (
+            <div className="checkbox">
+              <label>
+                <input
+                  type="checkbox"
+                  className="create-storage-class-form__checkbox"
+                  onChange={(event) => this.setStorageHandler('expansion', event)}
+                  checked={isExpansion}
+                  id="expansion-checkbox"
+                  aria-describedby="expansion-checkbox"
+                />
+                Allow user to expand the persistent volume claim.
```

Comment:
Checkbox labels shouldn't end with a period.

I'd probably say

```suggestion
                Allow persistent volume claims to be expanded
```